Create ui-c8y-1021-22-56-reintroduce-dashboard-time-range-and-integrate-global-date-context-with-dpt.md (#3116)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-56-reintroduce-dashboard-time-range-and-integrate-global-date-context-with-dpt.md

---
date: ""

> title: Reintroduce dashboard time range and integrate global date context with DPT (#7747) [GRAFT][release/y2025] (#8177)

Reviewer: ```suggestion
title: Restore the dashboard time range selector option and integrate global date context with the data points table widget
```

> title: Restore the dashboard time range selector option and integrate global date context with the data points table widget

Reviewer 2: ```suggestion
title: Restored dashboard time range selector option for "Data points table" widget
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61920
version: 1021.22.56
---

> Reintroduce dashboard time range and integrate global date context with DPT (#7747) [GRAFT][release/y2025] (#8177)

Reviewer: ```suggestion
In order to ensure the seamless operation of dashboards with global date context and the data point table widget, we have temporarily removed the "Dashboard time range" date selection option from the data point table widget. We are now restoring the "Dashboard time range" option to this widget.
```

> In order to ensure seamless operation of dashboards with global date context and the "Data point table" widget, the "Dashboard time range" date selection option had temporarily been removed from the "Data point table" widget. This option has now been restored for this widget.

Reviewer 2: ```suggestion
In order to ensure seamless operation of dashboards with global date context and the "Data point table" widget, the dashboard time range selection option had temporarily been removed from the "Data point table" widget. This option has now been restored for this widget.
```

> In order to ensure the seamless operation of dashboards with global date context and the data point table widget, we have temporarily removed the "Dashboard time range" date selection option from the data point table widget. We are now restoring the "Dashboard time range" option to this widget.

Reviewer 2: ```suggestion
In order to ensure seamless operation of dashboards with global date context and the "Data point table" widget, the "Dashboard time range" date selection option had temporarily been removed from the "Data point table" widget. This option has now been restored for this widget.
```
